2023/src/theme.ts

```diff
@@ -27,15 +27,36 @@ export const theme: DefaultTheme = {
   },
   fonts: {
     header: "Lato",
-    text: "Arial",
+    text: [
+      "Frutiger",
+      "Frutiger Linotype",
+      "Univers",
+      "Calibri",
+      "Gill Sans",
+      "Gill Sans MT",
+      "Myriad Pro",
+      "Myriad",
+      "DejaVu Sans Condensed",
+      "Liberation Sans",
+      "Nimbus Sans L",
+      "Tahoma",
+      "Geneva",
+      "Helvetica Neue",
+      "Helvetica",
+      "Arial",
+      "Hiragino Sans",
+      "sans-serif",
+    ]
+      .map(n => `"${n}"`)
+      .join(", "),
   },
   fontSizes: {
     hero: "6rem",
     title: "4rem",
     subTitle: "2.6rem",
   },
   breakpoints: {
-    mobile: "@media (max-width: 576px)",
```

Comment:
The breakpoints are a separate PR I think.

Reply:
You are right, just a sec and I correct it.

Reply:
fixed :)

Reply:
I don't think its fixed, as it is still part of this pr :sweat_smile:

Reply:
Sorry, I forgot to remove stashed changes 😢